app/src/lib/stores/app-store.ts

```diff
@@ -1826,6 +1826,28 @@ export class AppStore extends TypedBaseStore<IAppState> {
         this.statsStore.recordCoAuthoredCommit()
       }
 
+      const account = getAccountForRepository(this.accounts, repository)
```

Comment:
Now we will record a enterprise/dotcom commit so long as they are in a GH hosted repo.